src/models/User.ts

```diff
@@ -1,31 +1,68 @@
 import mongoose, { Document, Schema } from "mongoose";
+import bcrypt from "bcryptjs";
+
+const root = process.env.PROFILE_PICTURE_ROOT;
```

Comment:
We should indicate on the README what this variable should be. I would possibly argue that if this path is the same for everyone it shouldn't be a variable, but if this is going to be different when we build and deploy this app then it still makes sense to be a variable.